azure-devops/azext_devops/dev/repos/pull_request.py

```diff
@@ -99,7 +99,7 @@ def list_pull_requests(repository=None, creator=None, include_links=False, revie
 def create_pull_request(project=None, repository=None, source_branch=None, target_branch=None,
                         title=None, description=None, auto_complete=False, squash=False,
                         delete_source_branch=False, bypass_policy=False, bypass_policy_reason=None,
-                        merge_commit_message=None, reviewers=None, work_items=None,
+                        merge_commit_message=None, reviewers=None, work_items=None, is_draft=False,
```

Comment:
I think in general we are discouraging switch in our code.
The reason was that in create --is-draft will behave differently than --is-draft in update.
Just check with @geverghe once before pushing if we are fine with the difference in create/ update

Reply:
@gauravsaralMs In my understanding the reason for the change that time was having different paradigms in the same command for variables seemingly same like --is-draft being a bool, --is-xyz being a string. 

@geverghe 
The concern here is that --is-draft is a flag in create PR like --open and other flags in create whose presence/absence signifies true or false instead of explicit value. 
In update command it is string which can be true or false. 

**Reasoning:** Create Pr has multiple other boolean flags so --is-draft being a bool is consistent in the command scope. 
**Why can't we have same in update PR command?** Because then --is-draft will have a default value and it will become compulsory for user to provide it during every update so we do not override the current state of PR with the default value. e.g. A PR is published and  I want to update the title.  In the command --is-draft is true by default. 
The command az repos pr update --title "abc" should have sufficed but this will change my PR to draft mode.

Reply:
Copying the comment to workitem and merging the PR. Will discuss this in bug bash and close.